CHANGELOG.md

```diff
@@ -4,11 +4,16 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Helm: support labels in values.yaml ([#201]).
+
 ### Fixed
 
 - Respect `--watch-namespace` CLI argument ([#193]).
 
 [#193]: https://github.com/stackabletech/commons-operator/pull/193
+[#201]: https://github.com/stackabletech/commons-operator/pull/201
```

Comment:
note: This link also needs to be changed.

Reply:
done